crates/rpc/rpc-testing-util/src/trace.rs

```diff
@@ -159,6 +159,68 @@ impl<'a> std::fmt::Debug for TraceBlockStream<'a> {
     }
 }
 
+//This part is for testing.
+//
+//------------------------
+/// A utility to compare RPC responses from two endpoints.
+#[derive(Debug)]
+pub struct RpcComparer<'a> {
+    endpoint1: &'a str,
+    endpoint2: &'a str,
+}
```

Comment:
this should take two clients, ideally abstracted over jsonrpsee client traits

Reply:
> this should take two clients, ideally abstracted over jsonrpsee client traits

Change has been made

Hope this is good for you boss :)

Thank you for giving me those information ! I never tought it was better with this way !